packages/types/src/dedupeCheckIssues.ts

```diff
@@ -0,0 +1,30 @@
+export interface DedupeCheckIssues {
```

Comment:
let's create a new directory called `dedupe` and a new package with these types: `@pnpm/dedupe.types`

Reply:
Done! https://github.com/gluxon/pnpm/commit/23788d6553a60b430145f521dd66b965252f2162